zuul-core/src/main/java/com/netflix/zuul/context/CommonContextKeys.java

```diff
@@ -43,7 +43,12 @@
     public static final String USE_FULL_VIP_NAME = "use_full_vip_name";
     public static final String ACTUAL_VIP = "origin_vip_actual";
     public static final String ORIGIN_VIP_SECURE = "origin_vip_secure";
-    public static final String REMOTE_PORT = "remote_port";
+
+    /**
+     * The original client port reported to Zuul by a proxy running Proxy Protocol.
+     * Will only be set if both Zuul and the connected proxy are both using set to use Proxy Protocol.
+     */
+    public static final String PROXY_PROTOCOL_PORT = "proxy_protocol";
```

Comment:
I think the value should be the same as the variable name.

Reply:
thanks for catching that, updated